### ISSUE TITLE
Define CNV genotype strategy

### ISSUE THREAD

**Issue by ctsa** [PacificBiosciences/HiFiCNV#19]
HiFiCNV currently outputs `0/1` as a placeholder GT value for all cases. Set policy to either make this unknown or provide a consistent interpretation of copy number as GT.

**Comment by mrvollger** [comment]
Hi @ctsa, I have a related question here, I think. 

Do you have any plans to make the CNV caller hap aware if the input is a haplotagged bam from hiphase for example? Similar to how your pb-cpg-tools works.

I think it would make an excellent addition ;)

**Comment by ctsa** [comment]
Thanks Mitchell, Are you thinking of allele-specific copy number output for duplications? If so, we might be able to do it with the current MAF input. Would be interested in knowing more about the application you have in mind.

**Comment by mrvollger** [comment]
Yeah exactly. I want to explore if there are allele-specific dosage effects for certain genes in a cancer line we are studying.

An example might be: hap2 has gained an allele-specific variant in gene X, but also has a CN change of Y. Is the allele-specific variant having an impact on gene dosage beyond the CN change in the haplotype?

**Comment by ctsa** [comment]
Thanks, good to know about the case you're thinking of. We're probably a few steps away from what you describe due to generally limited support for somatic CNA calling. The most directly applicable of these is that we're limiting classifications of duplications to a simple "CN 5 or greater" state on the top end. This together with the lack of a purity/ploidy estimation/correction model mean we would have some ground to cover before we could make allele-specific CN very useful in a somatic context directly in HiFiCNV.

We have some separate tooling recommendations for this problem but these are moving pretty quickly, so let me connect you into a separate discussion to make sure I'm getting you the most up to date info.
